tutorials/advanced/transforms_tutorial.py

```diff
@@ -229,15 +244,13 @@
 # move, but the circle will remain fixed because it is not in `data`
 # coordinates and will always remain at the center of the axes.
 
-import matplotlib.patches as patches
-
 fig = plt.figure()
 ax = fig.add_subplot(111)
 x, y = 10*np.random.rand(2, 1000)
-ax.plot(x, y, 'go')  # plot some data in data coordinates
+ax.plot(x, y, 'go', alpha=0.2)  # plot some data in data coordinates
 
-circ = patches.Circle((0.5, 0.5), 0.25, transform=ax.transAxes,
-                      facecolor='yellow', alpha=0.5)
+circ = mpatches.Circle((0.5, 0.5), 0.25, transform=ax.transAxes,
+                      facecolor='blue', alpha=0.75)
```

Comment:
the continued line should start after the enclosing bracket:

~~~
circ = mpatches.Circle((0.5, 0.5), 0.25, transform=ax.transAxes,
                       facecolor='blue', alpha=0.75)
~~~

This is done wrong in multiple places in the example (not only your edits).